lib/common/options_display.c

```diff
@@ -301,17 +315,55 @@ add_option_metadata_xml(pcmk__output_t *out,
         desc_short = desc_long;
     }
 
-    // The standard requires a parameter type
-    CRM_ASSERT(option->type != NULL);
+    if (legacy) {
+        // This is ugly but it will go away at a major release bump
```

Comment:
likely 3.1 -- multiple distros ship pcs, so we have to wait until they all catch up, but it doesn't affect rolling upgrades so it doesn't have to be a major bump

Reply:
:'(